Chat/Parcer.h

```diff
@@ -15,8 +15,8 @@ enum ConstantSizes
     PREPARE_MESSAGE_SIZE = 5,   
```

Comment:
Maybe, it is better to create one constant with value 5 instead of 4 equal constants?